src/base/dig_printer.rs

```diff
@@ -24,7 +24,7 @@ impl<'a, Octs: AsRef<[u8]>> fmt::Display for DigPrinter<'a, Octs> {
         writeln!(
             f,
             ";; ->>HEADER<<- opcode: {}, rcode: {}, id: {}",
-            header.opcode().display_zonefile(false),
+            header.opcode().display_zonefile(DisplayKind::Simple),
```

Comment:
This looks really strange to me, but also the original as well: How can one display a zone file file an opcode? I guess it means `header.opcode().display_in_zonefile_presentation_format()` but that would be a bit long ;-)